code/nebula/neb.cpp

```diff
@@ -332,6 +332,19 @@ void neb2_init()
 	parse_modular_table("*-neb.tbm", parse_nebula_table);
 }
 
+// set the bits for poofs from a list of poof names
+void neb2_set_poof_bits(SCP_vector<SCP_string> list)
```

Comment:
This should be a reference (and probably a const reference) since there's no need to copy the entire list:
```suggestion
void neb2_set_poof_bits(const SCP_vector<SCP_string>& list)
```

Reply:
Done